lib/screens/item/widgets/main_view.dart

```diff
@@ -8,8 +8,10 @@ import 'package:flutter_slidable/flutter_slidable.dart';
 import 'package:hacki/blocs/blocs.dart';
 import 'package:hacki/config/constants.dart';
 import 'package:hacki/cubits/cubits.dart';
+import 'package:hacki/cubits/github/github_cubit.dart';
 import 'package:hacki/extensions/extensions.dart';
 import 'package:hacki/models/models.dart';
+import 'package:hacki/screens/item/widgets/github_repository_card.dart';
```

Comment:
export in `screens/item/widgets/widgets.dart`.

Reply:
I suggest you to please do the minor changes as per your flexibility because I am not proficient in these and might change some other thing